project/src/main/java/com/google/sps/Servlets/User.java

```diff
@@ -1,29 +1,18 @@
-// public class User{
-//   String email, username;
-//   int userID;
-  
-//   User(String email) {
-//     this.email = email;
-//   }
-
-//   public int getUserID() {
-//     return userID;
-//   }
+package com.google.sps.servlets;
 
-//   public String getUsername() {
-//     return username;
-//   }
+import java.util.List;
+import java.util.ArrayList;
 
-//   public List<int> getDocuments() {
-//     return Database.getUserDocumentIDs(getUserID());
-//     //make call to database and return list
-//   }
-
-//   public void addDocument(int docID) {
-//     //make call to database to add ID
-//   }
+public class User{
```

Comment:
if we can get the username from thee google auth API some of this may become redundant code, but for the time being it looks great

Reply:
Definitely agree, I will look into it next.